pkg/reconciler/labeler/v1/labels.go

```diff
@@ -30,9 +29,9 @@ import (
 	v1 "knative.dev/serving/pkg/apis/serving/v1"
 )
 
-// syncLabels makes sure that the revisions and configurations referenced from
+// SyncLabels makes sure that the revisions and configurations referenced from
 // a Route are labeled with route labels.
-func (c *Reconciler) syncLabels(ctx context.Context, r *v1.Route) error {
+func SyncLabels(r *v1.Route, cacc *Configuration, racc *Revision) error {
```

Comment:
Same here `targets ...Accessor` ?

Reply:
No, it needs the concrete type to List because it looks at Configuration.Status for example

Reply:
We can probably just downcast. But 🤷  doesn't matter much, I guess.

Reply:
We can totally part on this in V2. The next PR will basically be a big copy/paste of this so we can start making changes. If we modify this thing too much we're introducing risk which v1 is trying to freeze (despite the refactor). Let's reconsider these patterns in the future-looking version.